components/_twig-components/functions/pl_attach-library.function.php

```diff
@@ -5,5 +5,20 @@
  */
 
 $function = new Twig_SimpleFunction('attach_library', function ($string) {
-  return $string;
-});
+  // Get Library Name from string.
+  $libraryName = substr($string, strpos($string, "/") + 1);
+
+  // Search dist root directory for library.
+  $files = glob('dist/*/' . $libraryName . '/' . $libraryName . '*.js');
+  // Search dist nested directory for library.
+  if (empty($files)) {
+    $files = glob('dist/*/*/' . $libraryName . '/' . $libraryName . '*.js');
```

Comment:
Should we be using the dist directory here or the source directory?

Reply:
dist is correct - it's where the final minified file from the gulp scripts task is placed (and then a map file is created for debugging as well)